### PR TITLE
SMAGENT-1580: Increase daemonset limits for CPU and memory to be more…

### PR DESCRIPTION
**Pull request by anoop-sysd**
… realistic.

### CODE REVIEW

File: agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml [hunk 1]
```diff
@@ -65,13 +65,14 @@ spec:
         securityContext:
           privileged: true
         resources:
-          # Resources needed are subjective on the actual workload
-          # please refer to Sysdig Support for more info about it
+          # Resources needed are subjective on the actual workload,
+          # please contact Sysdig Support for tuning these numbers.
```
Comment (bryanseay) [inline agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml]: These are two seperate statements; you can't use a comma.  Also "subjective on" and "for tuning"  feel like odd phrases.

File: agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml [hunk 2]
```diff
@@ -65,13 +65,14 @@ spec:
...
           requests:
-            cpu: 100m
+            cpu: 600m
             memory: 512Mi
           limits:
-            memory: 1024Mi
+            cpu: 2000m
+            memory: 1536Mi
```
Comment (bryanseay) [inline agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml]: This is great but can you run this by Naveen? We talked about many different ways to do this, including having a small/medium/large example.

File: agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml (hunk 1 above)
Comment (bryanseay) [reply agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml]: At the end of the day, these are customer facing. Maybe AmyL should tweak it for us.

File: agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml (hunk 2 above)
Comment (balinaveen) [reply agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml]: Clearly, these larger numbers are better. But I'd recommend that we match the "small" config limits listed here: https://sysdig.atlassian.net/wiki/spaces/EN/pages/823984513/Agent+Resource+Allocation ?

We've started socializing the data in the above link with Support and having default numbers that don't match anything seems odd.

**Pull request merged**